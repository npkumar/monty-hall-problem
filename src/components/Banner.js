import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Typography } from "@material-ui/core";
import MeetingRoomIcon from "@material-ui/icons/MeetingRoom";
import NoMeetingRoomIcon from "@material-ui/icons/NoMeetingRoom";

const useStyles = makeStyles({
  root: {
    display: "flex",
    justifyContent: "center",
    alignItems: "center",
    color: "#52af77",
  },
});

const Banner = () => {
  const classes = useStyles();
  return (
    <Typography component="div" className={classes.root}>
      <NoMeetingRoomIcon fontSize="large" color="secondary" />
      <MeetingRoomIcon fontSize="large" />
      <NoMeetingRoomIcon fontSize="large" color="secondary" />
    </Typography>
  );
};

export default Banner;
