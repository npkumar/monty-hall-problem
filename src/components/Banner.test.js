import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";
import Banner from "./Banner";

describe("Banner", () => {
  afterEach(() => cleanup());

  it("renders", () => {
    const { asFragment } = render(<Banner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
