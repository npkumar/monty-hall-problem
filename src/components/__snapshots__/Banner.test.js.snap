// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders 1`] = `
<DocumentFragment>
  <div
    class="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-colorSecondary MuiSvgIcon-fontSizeLarge"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M11 11h-1v2h2v-1l9.73 9.73L20.46 23 14 16.54V21H3v-2h2V7.54l-4-4 1.27-1.27L11 11zm3 .49L5.51 3H14v1h5v12.49l-2-2V6h-3v5.49z"
      />
    </svg>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M14 6v15H3v-2h2V3h9v1h5v15h2v2h-4V6h-3zm-4 5v2h2v-2h-2z"
      />
    </svg>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-colorSecondary MuiSvgIcon-fontSizeLarge"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M11 11h-1v2h2v-1l9.73 9.73L20.46 23 14 16.54V21H3v-2h2V7.54l-4-4 1.27-1.27L11 11zm3 .49L5.51 3H14v1h5v12.49l-2-2V6h-3v5.49z"
      />
    </svg>
  </div>
</DocumentFragment>
`;
